lib/presentation/models/word_cloud/word_cloud_data.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/painting.dart';
+import 'dart:ui';
+
+class FlutterHashtag {
```

Comment:
Would be good for the class to have a more generic name like `Keyword`

Reply:
Addressed in commit 715784b21423ab1ba10a56f55d0253e659af2f69 & 21a427315cb7f8eb186c27121377715e5f76df66